### PR TITLE
Consistent norm computation

### PR DESCRIPTION
**Pull request by Yash-10**
From observations in #1433.

Thanks!

### CODE REVIEW

**Comment by Yash-10** [comment]
Trying to investigate the docs error...

**Comment by Yash-10** [comment]
It seems if the input array contains extra unwanted dimensions, it fails. `np.squeeze()` is not yet an option in `numba`, so maybe something else is needed.

(Maybe support to different types of norms, i.e. L1, L2, etc could also be kept just as in the case of `np.linalg.norm`)

**Comment by astrojuanlu** [comment]
I see that `norm(Ephem.from_body(Venus, flyby_1_time).rv()[0])` fails, because `Ephem.from_body(Venus, flyby_1_time).rv()[0].ndim == 2`. On the other hand, `norm(Ephem.from_body(Venus, flyby_1_time).rv()[0][0])` and `norm(Ephem.from_body(Venus, flyby_1_time).rv(flyby_1_time)[0])` succeed.

We want norms of vectors, not norm of matrices, so I think this failure is correct. It forces the user to put the vector in the correct shape though. Addmittedly, `Ephem.from_body(Venus, flyby_1_time).rv(flyby_1_time)[0]` is not a very convenient way of computing the position of a planet at a given time, we are probably overusing `Ephem` there. But I suggest we make the small change in the notebook and we continue.

File: src/poliastro/_math/linalg.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import numpy as np
+from numba import njit as jit
+
+
+@jit
+def norm_fast(arr):
```
Comment (astrojuanlu) [inline src/poliastro/_math/linalg.py:6]: Can we perhaps rename this to just `norm`, and import it as such? That way we minimize the lines changed too.

```suggestion
def norm(arr):
```

**Review by astrojuanlu — APPROVED** [review]
Looks perfect to me! When we fix the notebook that is failing, we can merge.

File: src/poliastro/_math/linalg.py (hunk 1 above)
Comment (astrojuanlu) [reply src/poliastro/_math/linalg.py:6]: (And, in modules where we have units-compatible `norm` and this function, we can do `from poliastro._math.linalg import norm as norm_fast`)

Comment (Yash-10) [reply src/poliastro/_math/linalg.py:6]: Yeah, this seems to be better!

**Comment by astrojuanlu** [comment]
Thanks a lot @Yash-10 !

**Pull request merged**